### PR TITLE
Added nginx and preview frame

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

**Review by cosletmihai — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -47,8 +47,9 @@ generate-indexes:
 	@ rm -rf ./output/classes/
 	@ rm -rf ./output/properties/
 
-
-
+generate-content:
```
Comment (costezki) [inline Makefile:50]: nice make target for the development phase

File: templates/content/templates/celex_classes/main.html [hunk 2]
```diff
@@ -3,14 +3,13 @@
...
 <h1>CELEX Classes</h1>
-{% set graph = conf.graphs.celex_classes %}
 
 {% set select_query =
 "PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 PREFIX euvoc: <http://publications.europa.eu/ontology/euvoc#>
 
 SELECT ?collection ?prefLabel
-FROM <"~graph~">
+FROM <~graph~>
```
Comment (costezki) [inline templates/content/templates/celex_classes/main.html:12]: oh! wow, this works now?

File: templates/content/templates/celex_classes/main.html [hunk 3]
```diff
@@ -22,15 +21,15 @@ <h1>CELEX Classes</h1>
     ?parent skos:member ?collection
   }
 } ORDER by ?order" %}
-{% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+{% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query, **conf.celexes).fetch_tabular() %}
```
Comment (costezki) [inline templates/content/templates/celex_classes/main.html:24]: I wonder why do we need to unpack the dict there, but I guess I did not yet reach to see the macro implementation yet.

**Review by costezki — APPROVED** [review]
It looks beautiful with those parameters inside the SPARQL query.  

Did you finally decide to use Tilda(~) as the paramater marker?

**Review by costezki — APPROVED** [review]
looks great

**Pull request merged**